images/linux/Ubuntu2004-Readme.md

```diff
@@ -100,7 +100,7 @@
 - SVN 1.13.0
 - Terraform 1.1.4
 - yamllint 1.26.3
-- yq 4.16.2
+- [yq](https://github.com/mikefarah/yq) 4.16.2
```

Comment:
You have to update `https://github.com/actions/virtual-environments/blob/92eeb55189d6faee20e7e2df1eb741fb5c140572/images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1#L292`, this is autogenerated documentation file.

Reply:
I think the default output is good:
```
$ ./yq_linux_amd64 --version
yq (https://github.com/mikefarah/yq/) version 4.17.2
```

Reply:
@wheelerlaw, Could you please make the changes?